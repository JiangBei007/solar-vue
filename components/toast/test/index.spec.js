// Import the mount() method from the test utils
// and the component you want to test
import { mount } from '@vue/test-utils'
import Counter from './../index'

describe('Counter', () => {
  test('renders correctly', () => {
    const wrapper = mount(Counter)
    expect(wrapper.element).toMatchSnapshot()
  })
})
