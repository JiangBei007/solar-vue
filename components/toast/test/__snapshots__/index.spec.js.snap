// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter renders correctly 1`] = `
<div
  style="display: table;"
>
  <div
    class="sv-toast sv-toast-text "
    name="sv-transition-fade"
    style="display: none;"
  >
    <div
      class=""
    />
    <div
      class="sv-toast-message"
    />
  </div>
</div>
`;
